README.md

```diff
@@ -177,6 +177,44 @@ var conversion = {
 client.leads.convert(conversion, callback);
 ```
 
+## Visitors
+
+```node
+// Update a visitor by id
+client.visitors.update({ id: '5435345', email: 'wash@serenity.io' }, callback);
+```
+
+```node
+// Find visitor by id or user_id
```

Comment:
`listBy` didn't really make sense here as user_id is a unique identifier.
I've updated the `find` method to handle `user_id` and `id`